src/current/v23.1/architecture/glossary.md

```diff
@@ -41,3 +41,220 @@ For more information on deployment options and guidelines on how to choose a dep
 {% include common/basic-terms.md %}
 
 For more information on CockroachDB Cloud, see [CockroachDB Cloud Architecture](../../cockroachcloud/architecture.html#architecture).
+
+## Spatial and GIS terms
```

Comment:
Thank you for adding the new section. I feel it's bit awkward that the h3s in the existing Glossary correspond to individual terms, whereas the h3s in the Spatial section each contain multiple terms. What do you think about just making all the h3s on the Glossary page into h4s -- in that case they won't show in the nav (I think) and the nav would be a little friendlier IMO.

Reply:
Fixed this by making H3s > H4s and moving the bulleted lists for spatial terms into H4s for consistency.